release-notes.md

```diff
@@ -1,5 +1,47 @@
 # Outpayce Checkout SDK - Release Notes
 
+### 4.4.0 - Supported until 15 May 2025
+- Added the ability for users to customize URLs for better connectivity and flexibility with the checkout system.
```

Comment:
This functionality is only available in the Core SDK, so I wouldn't mention it here